package.json

```
@@ -1,6 +1,6 @@
 {
   "name": "language-miniyaml",
-  "version": "0.0.1",
+  "version": "1.0.0",
```

```suggestion
  "version": "0.0.2",
```

I think version 1.0.0 is exaggerated. This was based on [yaml version 0.3.2](https://github.com/atom/language-yaml/blob/master/package.json#L3) and all the other https://github.com/github/linguist/blob/master/.gitmodules stay below `0.0.X` or `0.X.0` respectively.

---

One thing I've learned in recent years is that it is OK to get to version 1.0. It's not scary. You just need to set goals for version 1.0, 2.0 and maybe subversions and bump it when you achieve them.
Also I don't think that's how [semantic versioning](https://semver.org/) works, but OK.